DigiMeSDK/Core/Classes/Security/DMECrypto.m

```diff
@@ -249,6 +307,211 @@ -(NSData*)loadRSAKeyDataWithKeyClass:(CFTypeRef)keyClass  keyTagString:(NSString
     return nil;
 }
 
+- (NSData *)stripPublicKeyHeader:(NSData *)d_key
+{
+    // Skip ASN.1 public key header
+    if (d_key == nil)
+    {
+        return(nil);
+    }
+    
+    unsigned long len = [d_key length];
+    if (!len)
+    {
+        return(nil);
+    }
+    
+    unsigned char *c_key = (unsigned char *)[d_key bytes];
+    unsigned int idx = 0;
+    
+    if (c_key[idx++] != 0x30)
```

Comment:
Are there any comments we can add to describe what is going on in this method? 😄